### PR TITLE
OTC-344: Dialog button labels no longer hardcoded.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
https://openimis.atlassian.net/browse/OTC-344
Changes:
- Button labels added to resources and can now be translated.

### CODE REVIEW

**Review by delcroip — COMMENTED** [review]
please check my comment

File: app/src/main/assets/pages/FamilyPolicies.js [hunk 1]
```diff
@@ -57,29 +57,35 @@ $(document).ready(function () {
...
+                        click: function () {
+
+                            if (isOffline == 0 || isOffline == 2) {
+
+                                PolicyDeleted = parseInt(Android.DeleteOnlineData(PolicyId, 'PO'));
+                            }
+                            else {
+                                PolicyDeleted = Android.DeletePolicy(PolicyId);
+                            }
+                            if (PolicyDeleted == 1) {
```
[the remark this answers is not in the document]
Comment (wzglinieckisoldevelo) [reply app/src/main/assets/pages/FamilyPolicies.js:72]: I didn't write that code, I only changed the labels and did some testing. So personally I can't really answer your question. I think I added or removed tabulation in this section that's why git shows that I've made some changes to those lines.

[the remark this answers is not in the document]
Comment (malinowskikam) [reply app/src/main/assets/pages/FamilyPolicies.js:72]: @delcroip But all the ifs are inside the click handler, what do you mean?

[the remark this answers is not in the document]
Comment (delcroip) [reply app/src/main/assets/pages/FamilyPolicies.js:72]: the "click: function ()" seems to end line 80 (before 69) 
the folowing code was after the closure
`if (PolicyDeleted == 1) {
                            Android.ShowDialog(Android.getString('PolicyDeleted'));
                            window.open('FamilyPolicies.html?f=' + FamilyId + "&l=" + LocationId + '&r=' + RegionId + '&d=' + DistrictId, "_self");
                        }
                        else if (PolicyDeleted == -1) {
                            Android.ShowDialog(Android.getString('LoginToDeleteOnlineData'));
                        }
                        $(this).dialog("close");`

[the remark this answers is not in the document]
Comment (delcroip) [reply app/src/main/assets/pages/FamilyPolicies.js:72]: I see it is just an indentation artefact, I will merge it

**Review by delcroip — APPROVED** [review]

**Pull request merged**